dropshot/Cargo.toml

```diff
@@ -23,7 +23,8 @@ debug-ignore = "1.0.5"
 form_urlencoded = "1.2.1"
 futures = "0.3.30"
 hostname = "0.4.0"
-http = "0.2.9"
+http = "1"
```

Comment:
Is 1 right as the minimum required version? Just double-checking.

Reply:
Yep, not using anything newer than what was in 1.0.